assets/data0_21pure/progs/gametypes/ca.as

```diff
@@ -1131,17 +1137,28 @@ void GT_InitGametype()
         gametype.setTeamSpawnsystem( team, SPAWNSYSTEM_INSTANT, 0, 0, false );
 
     // define the scoreboard layout
+    String titles;
+    String layout;
+    if ( gametype.useSteamAuth )
+    {
+        titles += "AVATAR ";
```

Comment:
Maybe it's nicer to always allow `AVATAR and %a`, and just drop it in the client when there is no steam auth?